clang/test/CodeGen/aarch64-sve2p1-intrinsics/acle_sve2p1_cntp.c

```diff
@@ -2,7 +2,10 @@
 // REQUIRES: aarch64-registered-target
 // RUN: %clang_cc1 -triple aarch64-none-linux-gnu -target-feature +sve2p1 -S -O1 -Werror -emit-llvm -o - %s | FileCheck %s
 // RUN: %clang_cc1 -triple aarch64-none-linux-gnu -target-feature +sve2p1 -S -O1 -Werror -emit-llvm -o - -x c++ %s | FileCheck %s -check-prefix=CPP-CHECK
-// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -target-feature +sve2p1 -S -disable-O0-optnone -Werror -Wall -o /dev/null %s
```

Comment:
I didn't mean for this RUN line to be removed. Instead, I'd like to see this RUN line duplicated for both sve2p1 and sme2.